docs/apm.asciidoc

```diff
@@ -0,0 +1,207 @@
++
+[source,yaml]
+----
+apiVersion: apm.k8s.elastic.co/v1alpha1
+kind: ApmServer
+metadata:
+  name: apm-server-quickstart
+  namespace: default
+spec:
+  version: "7.2.0"
+  nodeCount: 1
+  secureSettings:
+    secretName: apm-secret-settings
+  config:
+    output:
+      elasticsearch:
+        password: "${ES_PASSWORD}"
+----
+
+==== Reference an already existing Elasticsearch cluster
+
+Now that we know how to use the APM keystore and customize the server configuration let see how to manually configure a secured connection to an already existing Elasticsearch cluster:
+
+. Create a secret with the Elasticsearch CA
++
+First we need to store the certificate authority of the Elasticsearch cluster in a secret:
++
+[source,yaml]
+----
+kubectl create secret generic es-ca --from-file=tls.crt=elasticsearch-ca.crt
+----
+In the above command the file `elasticsearch-ca.crt` must contain the CA certificate of the Elasticsearch cluster you want to use.
++
+. You can then mount this secret using the pod template and reference the file in the `config` of the APM server.
++
+Here is a complete example with a password stored in the Keystore as it is described in the previous section:
++
+[source,yaml]
+----
+apiVersion: apm.k8s.elastic.co/v1alpha1
+kind: ApmServer
+metadata:
+  name: apm-server-quickstart
+  namespace: default
+spec:
+  version: "7.2.0"
+  nodeCount: 1
+  secureSettings:
+    secretName: apm-secret-settings
+  config:
+    output:
+      elasticsearch:
+        hosts: ["my-own-elasticsearch-cluster:9200"]
+        username: elastic
+        password: "${ES_PASSWORD}"
+        protocol: "https"
+        ssl.certificate_authorities: ["/usr/share/apm-server/config/elasticsearch-ca/tls.crt"]
+  podTemplate:
+    spec:
+      containers:
+      - name: apm-server
+        volumeMounts:
+        - mountPath: /usr/share/apm-server/config/elasticsearch-ca
+          name: elasticsearch-ca
+          readOnly: true
+      volumes:
+      - name: elasticsearch-ca
+        secret:
+          defaultMode: 420
+          optional: false
+          secretName: es-ca # This is the secret that holds the Elasticsearch CA cert
+----
+
+==== TLS Certificates
+
+By default the operator manages a private CA and generates a self-signed certificate used to secure the communication between APM agents and the server.
+
+This behavior and the relevant configuration is identical to what is done for Elasticsearch and Kibana. You can refer to link:k8s-accessing-elastic-services.html#k8s-setting-up-your-own-certificate[this documentation] to understand how to use your own certificate to configure the TLS endpoint of the APM server.
+
+=== Connecting to the APM server
+
+==== APM Server service
+
+The APM server is exposed with a Service, you can refer to link:k8s-accessing-elastic-services.html[this documentation] to understand how to access it.
```

Comment:
As @thbkrkr pointed out recently I believe the official spelling is "APM Server" with captital s